pkg/lib/config/sms_gateway.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+type SMSGatewayTwilioCredentials struct {
```

Comment:
Change the prefix of alls struct in this file to `SMSGatewayEnvironment`, for example, `SMSGatewayEnvironmentTwilioCredentials`